kubernetes/cache/cache.go

```diff
@@ -149,6 +151,34 @@ func NewKialiCache() (KialiCache, error) {
 	kialiCacheImpl.k8sApi = istioClient.GetK8sApi()
 	kialiCacheImpl.istioApi = istioClient.Istio()
 
+	// Update SA Token
+	ticker := time.NewTicker(tokenExpireDuration)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				if newToken, err := kubernetes.GetKialiToken(); err != nil {
+					log.Errorf("Error updating Kiali Token")
```

Comment:
Maybe we need to add error details here too:)

Reply:
Thanks, @Xunzhuo ! I missed this one